compiler-core/src/language_server/tests/compilation.rs

```diff
@@ -70,26 +172,30 @@ fn compile_error_in_test() {
     io.test_module("app/error", "pub type Error {");
 
     let response = engine.compile_please();
+
     assert!(response.result.is_err());
     assert!(response.warnings.is_empty());
     assert!(response.compiled_modules.is_empty());
 
     drop(engine);
-    let actions = io.into_actions();
+
+    let actions_during_new = vec![
+        Action::DependencyDownloadingStarted,
+        Action::DownloadDependencies,
+        Action::DependencyDownloadingFinished,
+        Action::LockBuild,
+        Action::UnlockBuild,
+    ];
+    let actions_during_compile = vec![
+        Action::CompilationStarted,
+        Action::LockBuild,
+        Action::UnlockBuild,
+        Action::CompilationFinished,
+    ];
```

Comment:
Could you revert these changes that split the vector in two please? 🙏  And in the other test also.